renovate.json5

```diff
@@ -833,6 +825,14 @@
           '@types/xml-crypto',
         ],
       },
+      {
+        groupSlug: 'xml2js',
+        groupName: 'xml2js related packages',
+        packageNames: [
+          'xml2js',
+          '@types/xml2js',
+        ],
+      },
```

Comment:
No idea why this switched positions, but it probably follows the yarn position changes :woman_shrugging: